python/cudf/cudf/core/window/ewm.py

```diff
@@ -0,0 +1,200 @@
+        Not Supported
+    times : str, np.ndarray, Series, default None
+        Not Supported
+
+    Returns
+    -------
+    ``ExponentialMovingWindow`` object
+
+    Notes
+    -----
+    cuDF input data may contain both nulls and nan values. For the purposes
+    of this method, they are taken to have the same meaning, meaning nulls
+    in cuDF will affect the result the same way that nan values would using
+    the equivalent pandas method.
+
+    .. pandas-compat::
+        **cudf.core.window.ExponentialMovingWindow**
+
+        The parameters ``min_periods``, ``ignore_na``, ``axis``, and ``times``
+        are not yet supported. Behavior is defined only for data that begins
+        with a valid (non-null) element.
+
+        Currently, only ``mean`` is a supported method.
+
+    Examples
+    --------
+    >>> df = cudf.DataFrame({'B': [0, 1, 2, cudf.NA, 4]})
+    >>> df
+          B
+    0     0
+    1     1
+    2     2
+    3  <NA>
+    4     4
+    >>> df.ewm(com=0.5).mean()
+              B
+    0  0.000000
+    1  0.750000
+    2  1.615385
+    3  1.615385
+    4  3.670213
+
+    >>> df.ewm(com=0.5, adjust=False).mean()
+              B
+    0  0.000000
+    1  0.666667
+    2  1.555556
+    3  1.555556
+    4  3.650794
+    """
+
+    def __init__(
+        self,
+        obj,
+        com: float | None = None,
+        span: float | None = None,
+        halflife: float | None = None,
+        alpha: float | None = None,
+        min_periods: int | None = 0,
+        adjust: bool = True,
+        ignore_na: bool = False,
+        axis: int = 0,
+        times: str | np.ndarray | None = None,
+    ):
+        if (min_periods, ignore_na, axis, times) != (0, False, 0, None):
+            raise NotImplementedError(
+                "The parameters `min_periods`, `ignore_na`, "
+                "`axis`, and `times` are not yet supported."
+            )
+
+        self.obj = obj
+        self.adjust = adjust
+        self.com = get_center_of_mass(com, span, halflife, alpha)
+
+    def mean(self):
+        """
+        Calculate the ewm (exponential weighted moment) mean.
+        """
+        return self._apply_agg("ewma")
+
+    def var(self, bias):
+        raise NotImplementedError("ewmvar not yet supported.")
+
+    def std(self, bias):
+        raise NotImplementedError("ewmstd not yet supported.")
+
+    def corr(self, other):
+        raise NotImplementedError("ewmcorr not yet supported.")
+
+    def cov(self, other):
+        raise NotImplementedError("ewmcov not yet supported.")
+
+    def _apply_agg_series(self, sr, agg_name):
+        if not is_numeric_dtype(sr.dtype):
+            raise TypeError("No numeric types to aggregate")
+
+        # libcudf ewm has special casing for nulls only
+        # and come what may with nans. It treats those nulls like
+        # pandas does nans in the same positions mathematically.
+        # as such we need to convert the nans to nulls before
+        # passing them in.
```

Comment:
Can we make this comment clearer? What does "come what may" mean here, precisely?